src/main/java/org/apereo/openequella/adminconsole/launcher/ProxySettingsDialog.java

```diff
@@ -79,27 +79,32 @@ public ProxySettingsDialog(Frame frame, ProxySettings proxySettings) {
     usernameField = new JTextField();
     passwordField = new JPasswordField();
     hostField = new JTextField();
-    portModel = new SpinnerNumberModel(PROXY_DEFAULT, PROXY_MIN, PROXY_MAX, PROXY_STEP);
-    
+    portField = new JTextField();
```

Comment:
I made some changes here. Instead of using a  JTextfield, JFormattedTextField helps limit that only numbers are valid here.